releases/8.0.0.md

```diff
@@ -139,7 +140,8 @@ JavaScript:
 - `WorkflowHistoryView` moved from `arches.app.views.workflow_history` to `arches.app.views.api`.
 - The format of JavaScript files has changed. Please refer to [Upgrading an Arches project](#upgrading-an-arches-project).
 - The format of how some images are loaded in template files has changed. Please refer to [Upgrading an Arches project](#upgrading-an-arches-project).
-- Update any imports from `@primevue/themes` to `@primeuix/themes`.
+- `arches-project` management commands have been deprecated.
```

Comment:
Can you also remove the deprecation warning in handle() in the surviving command?

I think it would aid readability to say it was deprecated in 7.6 and removed in 8.0. Or just "following a deprecation in 7.6, the foobar was removed." or similar

Reply:
My bad, you already removed that file, nvm about the handle() part